content/community/raising-issues/Raising Issues.adoc

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
Please remove the "Labelling" section from this file, since now we have that information in separate files.